src/bin/cratesfyi.rs

```diff
@@ -582,6 +586,37 @@ impl DatabaseSubcommand {
                     .block_on(ctx.repository_stats_updater()?.backfill_repositories())?;
             }
 
+            Self::BackfillCrateOwnerKind => {
```

Comment:
I tested this and I don't think `CASE WHEN` can be used directly since `DEFAULT` expressions can't refer to other columns. I added it as a separate statement in the up-migration file.

Reply:
awesome, thanks!